### PR TITLE
[Feat] 1.1.2 Generate match bracket from teams

### PR DESCRIPTION
**Pull request by rosengrenen**

### CODE REVIEW

**Review by Noxemia — DISMISSED** [review]
looks cool

File: packages/backend/src/generate-bracket.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+interface Match {
+	id: string;
+	firstTeamId: string | null;
+	secondTeamId: string | null;
+	firstParentId: string | null;
+	secondParentId: string | null;
+	round: number;
+}
```
Comment (cajoho99) [inline packages/backend/src/generate-bracket.ts]: Shouldn't this use the already defined types?

**Review by cajoho99 — CHANGES_REQUESTED** [review]
Looks good but should probably merge with master and change the types before merging

**Review by cajoho99 — APPROVED** [review]
Ser superbra ut!

**Review by SantaPoro — APPROVED** [review]

**Pull request merged**